### PR TITLE
Shooter Nudge Commands

### PR DESCRIPTION
**Pull request by duane701**
A Shooter Nudge command.

### CODE REVIEW

**Comment by duane701** [comment]
Closing for a new branch.

File: src/Commands/NudgeShooter.h [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#ifndef NudgeShooter_H
+#define NudgeShooter_H
+
+#include "Commands/InstantCommand.h"
+
+class NudgeShooter : public InstantCommand {
+  double mNudge;
```
Comment (SkylerPIlot) [inline src/Commands/NudgeShooter.h]: Make this a private variable `private:`

File: src/Commands/SetShooter.cpp [hunk 2]
```diff
@@ -33,3 +35,11 @@ void SetShooter::Interrupted() {
```
Comment (SkylerPIlot) [inline src/Commands/SetShooter.cpp]: Get rid of this whitespace

File: src/Commands/SetShooter.h [hunk 3]
```diff
@@ -11,8 +13,13 @@ class SetShooter : public frc::Command {
   bool IsFinished();
   void End();
   void Interrupted();
+
+  double GetSpeed() const;
+  void SetSpeed(double speed);
 private:
 	double mSpeed;
```
Comment (SkylerPIlot) [inline src/Commands/SetShooter.h]: fix tabing here

File: src/Commands/SetShooter.h [hunk 4]
```diff
@@ -11,8 +13,13 @@ class SetShooter : public frc::Command {
   bool IsFinished();
   void End();
   void Interrupted();
+
+  double GetSpeed() const;
```
Comment (SkylerPIlot) [inline src/Commands/SetShooter.h:17]: follow the example of how functions are organized

**Review by SkylerPIlot — CHANGES_REQUESTED** [review]

File: src/Commands/SetShooter.h (hunk 4 above)
Comment (duane701) [reply src/Commands/SetShooter.h:17]: Functions should be organized alphabetically or  as in this case, by related functionality. That is the Getter and the Setter are next to each other.

Also, whitespace is great but we don't have to use every other line.

**Comment by duane701** [comment]
Closed in favor of #31

**Pull request closed without merging**